doc/developer_guide/coding-style.rst

```diff
@@ -155,6 +155,47 @@ retained.). For example::
     except KeyError as err:
         raise InternalError("Useful message here") from err
 
+
+Logging
+-------
+
+PSyclone supports the Python `logging` module, to help enable improved
+general messages throughout the code. By default the logging infrastructure
+is switched off, but can be enabled either through the PSyclone script
+options or programatically by::
+
+    import logging
+    logging.basicConfig(level=logging.LEVEL)
+
+where `logging.LEVEL` is the required level of those defined in the standard
+logging library.
+
+To use logging in a module, you can add::
+
+    import logging
+    logger = logging.getLogger(__name__)
+    logging.info("Example of a logging message.")
+
+at the top of a module to make the logger available everywhere in the file.
+This is standard practice for using logging in Python.
```

Comment:
I would delete these two sentences (part of it not making sense was due to my previous suggestion of add the logging.info as part of the code above - sorry)

Reply:
Added a comment saying that the logger can be initialised at the module scope instead to clear that up.